lib/napa/logger/configuration.rb

```diff
@@ -0,0 +1,36 @@
+module Napa
+  class Logger
+    class Configuration
+
+      def initialize(options = {})
+        @options = {}.tap do |o|
+          o[:format] = :basic if Napa.env.development?
+          o[:output] = [:stdout] if ENV['DYNO']
+        end
+
+        @options.merge!(options)
+      end
+
+      def format
+        # Allowed options: :basic, :yaml, :json
+        @options[:format] || :json
```

Comment:
This may be a premature optimization, but I wonder if there are cases in which you would want to specify different formats for the various output types. For example, using the basic format for the file logger and a JSON format for the STDOUT logger.